src/content/en/updates/2018/06/feature-policy.md

```diff
@@ -0,0 +1,336 @@
+<iframe src="https://example.com..." allow="fullscreen"></iframe>
+
+<!-- Equivalent to: -->
+<iframe src="https://example.com..." allow="fullscreen *"></iframe>
+
+<!-- Allow only iframe content on a particular origin to access the user's location. -->
+<iframe src="https://google-developers.appspot.com/demos/..."
+        allow="geolocation https://google-developers.appspot.com"></iframe>
+```
+
+Note: Frames inherit the policy settings of their parent page. If the page
+and iframe both specify a policy list, the more restrictive allow list will
+be used. See [Inheritance rules](#inheritancerules).
+
+#### What about the existing iframe attributes? {: #legacy }
+
+Some of the [features controlled by Feature Policy](#list) have an existing
+ attribute to control their behavior. For example, `<iframe allowfullscreen>`
+is an attribute that allows the iframe's content to use the
+`HTMLElement.requestFullscreen()` API. There's also the `allowpaymentrequest` and
+`allowusermedia` attributes for allowing the
+[Payment Request API](/web/fundamentals/payments/) and `getUserMedia()`,
+respectively.
+
+Try to **use the `allow` attribute instead of these old
+attributes** where possible. For cases where you need to support backwards
+compatibility using the `allow` attribute with an equivalent legacy attribute
+is perfectly fine (e.g. `<iframe allowfullscreen allow="fullscreen">`).
+Just note that the more restrictive policy wins. For example, the following
+iframe would not be allowed to enter full screen because
+`allow="fullscreen 'none'"` is more restrictive than `allowfullscreen`:
+
+```html
+<!-- Blocks full screen access if the browser supports feature policy. -->
+<iframe allowfullscreen allow="fullscreen 'none'" src="...">
+```
+
+## Controlling multiple policies at once {: .multiple }
+
+Several features can be controlled at the same time by sending the HTTP header
+with a `;` separated list of policies:
+
+```
+FeaturePolicy: unsized-media 'none'; geolocation 'self', https://example.com; camera *;
+```
+
+or by sending a separate header for each policy:
+
+```
+FeaturePolicy: unsized-media 'none'
+FeaturePolicy: geolocation 'self', https://example.com
+FeaturePolicy: camera *;
+```
+
+This example would do the following:
+
+- Disallows the use of `unsized-media` for all browsing contexts.
+- Disallows the use of `geolocation` for all browsing contexts except for the
+page's own origin and `https://example.com`.
+- Allows `camera` access for all browsing contexts.
+
+**Example** - setting multiple policies on an iframe
+
+```html
+<!-- Blocks the iframe from using the camera and microphone
+     (if the browser supports feature policy). -->
+<iframe allow="camera 'none'; microphone 'none'">
+```
+
+
+## JavaScript API {: #js }
+
+Heads up: While Chrome 60 added support for the `FeaturePolicy` HTTP header
+and the `allow` attribute on iframes. The [JavaScript API][jsapi] is still
```

Comment:
This should not be the start of a new sentence. Previous clause ends with a comma.